scripts.py

```diff
@@ -633,6 +633,10 @@ def pagination(self, lane):
         for pagenum in range(0, self.pages):
             yield page
             page = page.next_page
+            if not page:
+                # There aren't enough books to fill `self.pages`
+                # pages. Stop working.
+                break
```

Comment:
This is an unrelated problem I found when running `bin/cache_opds_lane_facets` to force out problems with the code. If you ask to cache more pages than there are books in the lane, it was crashing.

I don't think this is a big deal since cache_opds_lane_facets hasn't been run on a regular basis for quite a while; it's more efficient to generate paginated feeds as people request them.